bots/faq_bot.py

```diff
@@ -121,6 +122,15 @@ async def on_turn(self, turn_context: TurnContext):
 
         await self.conversation_state.save_changes(turn_context)
 
+    # text is alread trimmed of whitespaces
+    def _clean_question(self, text: str):
+        clean_text = text
+        # remove slack mention
+        if text.startswith("@taiwan-bot"):
+            clean_text = ' '.join(text.split()[1:])
```

Comment:
I'd recommend using str.replace to remove any instance of @taiwan-bot anywhere. One doesn't have to tag the bot at the beginning of the message.

We can then remove white space with str.strip and replacing double spaces with single space (if that's necessary).

Reply:
@matrinox yes, https://github.com/taiwangoldcard/taiwan-bot/pull/38/commits/5d27029c6908960c6242963601c44190975c427c

for the whitespace, I think it's unecessary for now